runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/test/aws/smithy/kotlin/runtime/http/engine/okhttp/ByteChannelRequestBodyTest.kt

```diff
@@ -15,10 +15,9 @@ import kotlinx.coroutines.test.runTest
 import okio.Buffer
 import okio.BufferedSink
 import org.junit.jupiter.api.Test
+import java.io.IOException
 import kotlin.coroutines.EmptyCoroutineContext
-import kotlin.test.assertEquals
-import kotlin.test.assertFalse
-import kotlin.test.assertTrue
+import kotlin.test.*
```

Comment:
question: should we generally avoid * imports?

Reply:
No we usually just let the IDE optimize imports in the runtime.